src/CommunityToolkit.Maui.Core/Views/DrawingView/Service/DrawingViewService.windows.cs

```diff
@@ -0,0 +1,170 @@
+﻿using Windows.Storage.Streams;
+using Windows.UI.Input.Inking;
+using Microsoft.Graphics.Canvas;
+using Microsoft.Maui.Platform;
+
+namespace CommunityToolkit.Maui.Core.Views;
+
+/// <summary>
+/// Drawing view service
+/// </summary>
+public static partial class DrawingViewService
+{
+	/// <summary>
+	/// Get image stream from lines
+	/// </summary>
+	/// <param name="lines">Drawing lines</param>
+	/// <param name="imageSize">Image size</param>
+	/// <param name="backgroundColor">Image background color</param>
+	/// <returns>Image stream</returns>
+	public static Stream GetImageStream(IList<ILine>? lines,
+		Size imageSize,
+		Color backgroundColor)
+	{
+		if (lines == null)
+		{
+			return Stream.Null;
+		}
+
+		var image = GetImageInternal(lines, backgroundColor);
+		if (image == null)
+		{
+			return Stream.Null;
+		}
+
+		using (image)
+		{
+			var fileStream = new InMemoryRandomAccessStream();
+			image.SaveAsync(fileStream, CanvasBitmapFileFormat.Jpeg).GetAwaiter().GetResult();
+
+			var stream = fileStream.AsStream();
+			stream.Position = 0;
+
+			return stream;
+		}
+	}
+
+	/// <summary>
+	/// Get image stream from points
+	/// </summary>
+	/// <param name="points">Drawing points</param>
+	/// <param name="imageSize">Image size</param>
+	/// <param name="lineWidth">Line Width</param>
+	/// <param name="strokeColor">Line color</param>
+	/// <param name="backgroundColor">Image background color</param>
+	/// <returns>Image stream</returns>
+	public static Stream GetImageStream(IList<Point>? points,
+		Size imageSize,
+		float lineWidth,
+		Color strokeColor,
+		Color backgroundColor)
+	{
+		if (points == null || points.Count < 2)
+		{
+			return Stream.Null;
+		}
+
+		var image = GetImageInternal(points, lineWidth, strokeColor, backgroundColor);
+		if (image == null)
+		{
+			return Stream.Null;
+		}
+
+		using (image)
+		{
+			var fileStream = new InMemoryRandomAccessStream();
+			image.SaveAsync(fileStream, CanvasBitmapFileFormat.Jpeg).GetAwaiter().GetResult();
+
+			var stream = fileStream.AsStream();
+			stream.Position = 0;
+
+			return stream;
+		}
+	}
+
+	static CanvasRenderTarget? GetImageInternal(ICollection<Point> points,
+		float lineWidth,
+		Color lineColor,
+		Color backgroundColor)
+	{
+		if (points.Count == 0)
```

Comment:
If you want we can use pattern matching to check for constant values
```suggestion
		if (points.Count is 0)
```